onconnect/commands.py

```diff
@@ -55,48 +59,77 @@ async def _channel(self, ctx, *, channel: Optional[discord.TextChannel] = None):
         **Arguments:**
         - `[channel]` - Is where you set the event channel. Leave it blank to disable.
         """
+        embed_requested = await ctx.embed_requested()
         if channel:
-            if not channel.permissions_for(ctx.guild).manage_webhooks:
-                await ctx.send(
-                    "I do not have the `manage_webhooks` permission in {}.".format(
-                        channel.mention
-                    )
+            await self.config.statuschannel.set(channel.id)
+            log.info(f"Status Channel set to {channel} ({channel.id})")
+            if embed_requested:
+                embed = discord.Embed(
+                    title="Setting Changed",
+                    description=f"Events will now be sent to {channel.mention}.",
+                    colour=await ctx.embed_colour(),
                 )
+                await self.maybe_reply(ctx=ctx, embed=embed)
             else:
-                await self.config.statuschannel.set(channel.id)
-                await ctx.send(f"Event is now set to {channel.mention}")
+                await self.maybe_reply(
+                    ctx=ctx, message=f"Events will now be sent to {channel.mention}."
+                )
 
         elif await self.config.statuschannel() is not None:
-            msg = await ctx.send("Are you sure you want to disable events?")
+            msg = await ctx.maybe_send_embed("Are you sure you want to disable events?")
             start_adding_reactions(msg, ReactionPredicate.YES_OR_NO_EMOJIS)
+
             pred = ReactionPredicate.yes_or_no(msg, ctx.author)
             try:
                 await self.bot.wait_for("reaction_add", check=pred, timeout=30)
             except asyncio.TimeoutError:
-                await ctx.send("You took too long to response, cancelling.")
+                await self.maybe_reply(
+                    ctx=ctx,
+                    message="You took too long to respond, cancelling.",
+                    mention_author=True,
```

Comment:
I think mentions is best off by default. if people like pings, maybe a settings for it to change.

Reply:
Alright, good point there.